src/FSharp.Data.GraphQL.Shared/Errors.fs

```diff
@@ -129,48 +140,71 @@ type GQLProblemDetails = {
 
     static member internal CreateWithKind (message, kind : ErrorKind, ?path) = {
         Message = message
+        Exception = ValueNone
         Path = path |> Skippable.ofOption
         Locations = Skip
         Extensions = Dictionary<string, obj> 1 |> GQLProblemDetails.SetErrorKind kind |> Include
     }
 
-    static member Create (message, ?extensions : IReadOnlyDictionary<string, obj>) = {
+    static member Create (message, extensions : IReadOnlyDictionary<string, obj>) = {
         Message = message
+        Exception = ValueNone
         Path = Skip
         Locations = Skip
-        Extensions = extensions |> Skippable.ofOption
+        Extensions = Include extensions
     }
 
-    static member Create (message, extensions) = {
+    static member Create (message, [<Optional>] extensions) = {
         Message = message
+        Exception = ValueNone
         Path = Skip
         Locations = Skip
         Extensions = extensions
     }
 
-    static member Create (message, path, ?extensions : IReadOnlyDictionary<string, obj>) = {
+    static member CreateOfException (message : string, ex : Exception, [<Optional>] path : FieldPath Skippable, [<Optional>] extensions : IReadOnlyDictionary<string, obj> Skippable) = {
```

Comment:
Shouldn't we call it just `Create`? As in my mind if you emphasize that it is `CreateOfException` it is something like you read the Exception fields into the new object.
While adding just an input parameter have no additional cognitive load

Reply:
@njlr @nikhedonia @cmeeren @pkese any thoughts?

Reply:
Ok. It's just that I tend to avoid overloads, because I like to have a glance at the code and know exactly which function/method is being called 😄 .
But yeah, I can rename it to `Create`. This way it blends in with the other overloads.

Reply:
I don't use this library yet, but sure, I definitely prefer an overloads in these cases, for several reasons I don't care getting into at the moment.